app/components/profile/profileView.js

```diff
@@ -363,6 +363,9 @@ class ProfileView extends React.Component {
                     handleAddition={this.handleAdditionSkill}
                     handleDrag={this.handleDragSkill}
                     placeholder="Add skills"
+                    allowDeleteFromEmptyInput={false}
+                    handleInputBlur={this.handleInputBlur}
+                    autofocus={false}
```

Comment:
If dont have line 366, if user press delete when input text is empty, it will delete a tag.
Line 367 is suppose to make the suggestions disappear when input text lose focus but doesnt work.
Line 368 is to make it not be focus straightaway when in edit mode